python/spacewalk/spacewalk-backend.spec

```
@@ -286,6 +286,10 @@ do
 	sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!/usr/bin/python3=' $i;
 done
 
+%if !0%{?is_opensuse} && 0%{?sle_version}
+sed -i 's/PRODUCT_NAME = "Uyuni"/PRODUCT_NAME = "SUSE Manager"/' common/rhnConfig.py
```

A bit nitpicky, but maybe make this line bit more strict?
```suggestion
sed -i 's/^PRODUCT_NAME = "Uyuni"/PRODUCT_NAME = "SUSE Manager"/' common/rhnConfig.py
```